src/Build.UnitTests/BackEnd/BuildManager_Tests.cs

```diff
@@ -3805,7 +3805,7 @@ public void OutOfProcFileBasedP2PBuildSucceeds()
         /// not reload it from disk but instead fully utilize the entire translate project instance state
         /// to do the build.
         /// </summary>
-        /// <param name="shouldSerializeEntireState">Serializes the entire project instance state if true.</param>
+        /// <param name="shouldSerializeEntireState"><see langword="true"/> to serialize the entire project instance state; otherwise, <see langword="false"/>.</param>
```

Comment:
💡 In the future, note that you can insert the markup text quickly by the following steps:

1. Type `true` (or `false`)
2. Press <kbd>Ctrl</kbd>+<kbd>Space</kbd>

Reply:
I had no idea, that is really neat.